### DISCUSSION TITLE
Request to add the Arc browser to the browser matrix

### DISCUSSION THREAD

**Discussion by rsumner-godaddy** [privacytests/privacytests.org#177] [Ideas]
Can we get the Arc browser ( https://arc.net/ ) for desktop and iOS added to the browser matrix?

**Comment by Uri-22** [comment]
Any updates on this?

**Comment by vrsko** [comment]
Arc Browser is now also available for Windows. Are you planning to add it?

**Comment by sma33in** [comment]
upvoted
